front/lib/resources/content_fragment_resource.ts

```diff
@@ -582,7 +582,7 @@ export async function renderContentFragmentForModel(
         contentType,
         excludeImages,
         fileId,
-        forceFullCSVInclude: message.snippet != null, // JIT
+        forceFullCSVInclude: message.snippet != null, // If there is a snippet, it means that JIT was used, therefor if we are rendering the content fragment: we want to include the full content.
```

Comment:
please respect 100 columns wraps :pray:

Reply:
That must be my new editor messing up. Sorry about that.

Reply:
Actually, TIL comments are not affected by printWidth setting of prettier (because they can contains urls etc.. it's intentionnal from the prettier team).

I never manually wrapped anything, did you ?

Reply:
I generally wrap comments and tweak long strings